tools/cloud-build/daily-tests/tests/slurm-v5-hpc-centos7.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+deployment_name: "cos-v5-{{ build }}"
```

Comment:
More nits, sorry, cos usually stands for container-optimized os... you got me confused here for a sec. :)

Reply:
No problem! I was unaware. I did want it to somehow reflect that it's centos and not ubuntu, so I've updated it to "cent-v5-...".